lib/tabs/KTabs.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+
+  <div>
+    <KTabsList
+      v-model="activeTabId"
+      v-bind="{ tabsId, tabs, ariaLabel, ariaLabelledBy, color, colorActive, backgroundColor, hoverBackgroundColor, appearanceOverrides, appearanceOverridesActive, enablePrint }"
+    >
+      <!-- provide the same `tab` slot and send its properties further up from `KTabsList` -->
+      <template #tab="{ tab, isActive }">
+        <!-- @slot Optional slot for tab labels. Exposes `tab` object and `isActive` boolean. -->
+        <slot
+          name="tab"
+          :tab="tab"
+          :isActive="isActive"
+        >
+        </slot>
+      </template>
+    </KTabsList>
+
+    <KTabsPanel
+      :tabsId="tabsId"
+      :activeTabId="activeTabId"
+      :style="{ margin: '16px 0' }"
+    >
+      <!-- send the default slot and named slots to `KTabsPanel` which will then process them -->
+      <!-- @slot An alternative to using slots named by tabs' IDs. -->
+      <slot></slot>
+      <template v-for="(_, slot) in $slots" #[slot]>
```

Comment:
Is this newer Vue syntax? I haven't seen it before. Or maybe just the first time something like this was needed

Reply:
If you mean `#[slot]` part, then yes, it's available from 2.6.0. It's a mix of [Dynamic Slot Names](https://v2.vuejs.org/v2/guide/components-slots.html#Dynamic-Slot-Names) and [Named Slots Shorthand](https://v2.vuejs.org/v2/guide/components-slots.html#Named-Slots-Shorthand).